apps/chatbot/src/modules/settings.py

```diff
@@ -0,0 +1,92 @@
+import os
+import json
+import yaml
+from pathlib import Path
+from pydantic_settings import BaseSettings
+
+from src.modules.utils import get_ssm_parameter
+
+CWF = Path(__file__)
+ROOT = CWF.parent.parent.parent.absolute().__str__()
+PARAMS = yaml.safe_load(open(os.path.join(ROOT, "config", "params.yaml"), "r"))
+PROMPTS = yaml.safe_load(open(os.path.join(ROOT, "config", "prompts.yaml"), "r"))
+
+
+GOOGLE_SERVICE_ACCOUNT = get_ssm_parameter(
+    os.getenv("CHB_AWS_SSM_GOOGLE_SERVICE_ACCOUNT")
+)
+if GOOGLE_SERVICE_ACCOUNT is None:
+    with open(os.path.join(ROOT, ".google_service_account.json"), "r") as file:
+        GOOGLE_JSON_ACCOUNT_INFO = json.load(file)
+else:
+    GOOGLE_JSON_ACCOUNT_INFO = json.loads(GOOGLE_SERVICE_ACCOUNT)
+
+
+class ChatbotSettings(BaseSettings):
+    """Settings for the chatbot application."""
+
+    # api keys
+    aws_access_key_id: str = os.getenv("CHB_AWS_ACCESS_KEY_ID")
+    aws_default_region: str = os.getenv("CHB_AWS_DEFAULT_REGION")
+    aws_endpoint_url: str = os.getenv("CHB_AWS_SSM_ENDPOINT_URL")
+    aws_secret_access_key: str = os.getenv("CHB_AWS_SECRET_ACCESS_KEY")
+    google_api_key: str = get_ssm_parameter(
+        name=os.getenv("CHB_AWS_SSM_GOOGLE_API_KEY"),
+        default=os.getenv("CHB_AWS_GOOGLE_API_KEY"),
+    )
+    google_service_account: dict = GOOGLE_JSON_ACCOUNT_INFO
+    strapi_api_key: str = get_ssm_parameter(
+        os.getenv("CHB_AWS_SSM_STRAPI_API_KEY"), os.getenv("CHB_STRAPI_API_KEY", "")
+    )
+    langfuse_host: str = os.getenv("CHB_LANGFUSE_HOST")
```

Comment:
Since the evaluator is now responsible to write into langfuse is  this variable still needed?